osis_document/models.py

```
@@ -126,6 +127,51 @@ def save(self, force_insert=False, force_update=False, using=None, update_fields
         assert self.metadata.get('hash')
         super().save(force_insert, force_update, using, update_fields)
 
+    def get_file(self, modified=False):
+        if modified:
+            try:
+                return self.modified_upload.file
+            except Upload.modified_upload.RelatedObjectDoesNotExist:
+                pass
```

Pour éviter le pattern try... except pass..
Tu peux utiliser contextlib.suppress (cf. https://docs.python.org/3/library/contextlib.html#contextlib.suppress)